Example/AppDelegate.swift

```diff
@@ -17,10 +17,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             window!.rootViewController = UIViewController()
         }
         
+        listMapboxFrameworks()
```

Comment:
This is convenience method which allows to quickly find out what versions of Mapbox frameworks are used in Example app. Since `AppDelegate` is main point of entry I think it's acceptable to add this method here, also not sure it's worth moving it to another utils entity. Do you have any other ideas about place for this method?

Reply:
does it make sense in that case use `NSLog` instead of `print` to see those versions in the device log?